packages/network/src/createNetwork.ts

```diff
@@ -29,14 +29,16 @@ export async function createNetwork(initialConfig: NetworkConfig) {
 
   // Create signer
   const signer = computed<Signer | undefined>(() => {
-    const privateKey = config.privateKey;
     const currentProviders = providers.get();
+    if (config.provider.externalProvider) return currentProviders.json.getSigner();
+    const privateKey = config.privateKey;
     if (privateKey && currentProviders) return createSigner(privateKey, currentProviders);
   });
 
   // Get address
+  const initialConnectedAddress = config.provider.externalProvider ? await signer.get()?.getAddress() : undefined;
   const connectedAddress = computed(() =>
-    config.privateKey ? new Wallet(config.privateKey).address.toLowerCase() : undefined
+    config.privateKey ? new Wallet(config.privateKey).address.toLowerCase() : initialConnectedAddress
```

Comment:
i'm assuming the address we get from `signer.getAddress()` is checksummed? If we we need to normalize to lowercase here and also pass the non-lowercased address in `connectedAddressChecksummed`

Reply:
Yup it does: https://docs.ethers.io/v5/api/utils/address/#utils-getAddress